bin/generate_fake_data.rb

```diff
@@ -0,0 +1,111 @@
+#!env ruby
+
+require "factory_bot"
+require "faker"
+require_relative "../spec/fixtures/organizations"
+require_relative "../spec/fixtures/collections"
+
+FactoryBot.find_definitions
+
+Services.register(:ht_organizations) { mock_organizations }
+Services.register(:ht_collections) { mock_collections }
+
+OUTDIR = "/usr/src/app/testdata"
+MAX_OCN = 100000
+
+def random_year
+  Faker::Date.between(from: Date.parse("1850-01-01"), to: Date.today).year
+end
+
+def warn_exists(outfile)
+  if(File.exist?(outfile))
+    puts "#{outfile} already exists, not creating"
+    true
+  else
+    false
+  end
+end
+
+
+# For now, only single-part monographs
```

Comment:
The general idea here is to generate enough data to be somewhat realistic; using a maximum OCN of the same magnitude as the number of items we're generating ensures that we have htitems and holdings that are actually in the same cluster. We could reduce the maximum number of OCNs to get more holdings/htitems in a cluster, if we wanted. Note that we're only currently generating spms -- I am mostly interested in testing data loading, and most of the logic around data loading doesn't care at all about format.